test/unittest/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ include_directories(${SYCLBLAS_TEST} ${BLAS_INCLUDE_DIRS})
 # compiling tests
 set(SYCL_UNITTEST_SRCS
   # Blas 1 tests
+  ${SYCLBLAS_UNITTEST}/blas1/blas1_asum_test.cpp
```

Comment:
Do we want to enable this test given that we are not enabling the rest?

Reply:
The more tests we enable, the better. I'm enabling asum because I used it to fix the issue with negative increments, and we want to keep a test checking that it works properly. For the rest, we should enable all of them.